### ISSUE TITLE
Size of LyCORIS/Diag-OFT

### ISSUE THREAD

**Issue by iqddd** [bmaltais/kohya_ss#1789]
Why does the size of the LoRa weights (in MB) decrease with increasing Network Rank (DIM) when set to LyCORIS/Diag-OFT type?
For example, with DIM = 64 and Conv DIM = 64 the weights are 22.8 MB. And with DIM = 256 and Conv DIM = 256 it is only 11 MB.

**Comment by bmaltais** [comment]
Good question. I suggest you reach out to the author directly on his repo: https://github.com/KohakuBlueleaf/LyCORIS

**Comment by ScrapWare** [comment]
nerworks.oft and lycoris diag-oft is same, output same on oft calculate. but lycoris used another methods for as another output. Probably oft and ia3 is calculation finally 1dim these have different size to model dims with mlp, f[12], and others so Oft got lite weight. lora keys updown is fixed only 320,640,1280, and also less dims on text encoder. This issue is could simply solving, simply dim stacking by torch.cat.
